### ISSUE TITLE
Move from TypeORM to Prisma

### ISSUE THREAD

**Issue by mishraomp** [bcgov/quickstart-openshift#1442]
https://github.com/Kononnable/typeorm-model-generator/issues/329
```
I don't think typeorm should be considered as option for new projects other than prof of concepts or some workshop exercises. Typeorm is in a bad place. It's been like that for few years and I don't believe this will change any time soon(if ever). There are even some long standing bugs which can result in data loss. Some of the fixes are blocked by inability to introduce breaking changes - next branch(planned 0.3.0 version) has been with us for 3 years now and yet there are no concrete plans on releasing it or throwing that code away.
I've decided to mark this project as maintenance only to make sure that any newcomers think twice before choosing typeorm for their ORM.
```
TypeORM does not sound good and does not have good model generators, it is better to move to Prisma.
https://www.prisma.io/

thoughts @DerekRoberts ?

**Comment by DerekRoberts** [comment]
@mishraomp That's an excellent decision.

**Comment by mishraomp** [comment]
here is how to integrate with Prisma when using Nest
https://docs.nestjs.com/recipes/prisma

**Comment by mishraomp** [comment]
https://github.com/notiz-dev/nestjs-prisma-starter

**Comment by mishraomp** [comment]
Already done
